Google.Api.Generator.Tests/ProtoTests/BasicLro/BasicLroClient.cs

```diff
@@ -0,0 +1,74 @@
+﻿using gax = Google.Api.Gax;
+using gaxgrpc = Google.Api.Gax.Grpc;
+using lro = Google.LongRunning;
+using sys = System;
+using st = System.Threading;
+using stt = System.Threading.Tasks;
+
+namespace Testing.Basiclro
+{
+    /// <summary>Settings for a <see cref="BasicLroClient"/>.</summary>
+    public sealed partial class BasicLroSettings : gaxgrpc::ServiceSettingsBase
```

Comment:
I haven't checked - I assume this basically looks like the code we currently generate? (For both settings and the client.)
Anything you'd like me to look at carefully?

Reply:
Yes, it's all the same :)